### ISSUE TITLE
Add meaningful labels to UI elements such as "disable" toggles

### ISSUE THREAD

**Issue by hk-vmg** [screwdriver-cd/screwdriver#2436]
**What happened**:
On the Options page, the toggle buttons for disabling jobs are given generic ember "ID" and "aria-owns" attribute values like `ember86-x-toggle`. This means that screen readers/accessibility and scripting cannot find them easily.

**What you expected to happen**:
Please add an ID based on the job name, for example `pull-request-enabled-toggle`, `jobFoo-enabled-toggle` or similar.

**How to reproduce it**:
